README.md

```diff
@@ -87,13 +85,15 @@ flowchart LR;
 In order to use this workflow, you must already have completed the following steps:
 
 1. Created a Synapse project shared with challenge participants.
+1. Grant the DPE team `Administrator` privileges to said project.
 1. Created an evaluation queue within the Synapse project.
 1. One or more Docker images have already been submitted to your evaluation queue.
 1. Created a submission view that includes the `id` and `status` columns.
```

Comment:
Can there be multiple Evaluation Queues in the SubmissionView?  Or does the workflow only expect one Queue?  Does this detail matter?

Reply:
The detail doesn't matter, because the DAG will pull in whatever submissions show up in the SubmissionView, regardless of what evaluation queues correspond to it. So long as ORCA-SERVICES has visibility of the View and the ability to modify the submissions' annotations (so admin of the Evaluation Queue), it should be fine.

Reply:
OK good to know from our end!